hugo/content/resources/_index.md

```diff
@@ -24,6 +24,20 @@ Each of these tools will help your team collect and visualize <a href="/guides/d
 
 -----
 
+<section>
+    <article>
+    <a href="https://github.com/liatrio/backstage-dora-plugin" target="_blank">
+        <img src="img/liatrio.png" width="60" height="60">
+    </a>
+    <aside>
+    <h4 class="source-available-project"><a href="https://github.com/liatrio/backstage-dora-plugin" target="_blank">Liatrio backstage-dora-plugin</a> <a class="google-material-icons" href="https://github.com/liatrio/backstage-dora-plugin" target="_blank" aria-label="Liatrio backstage-dora-plugin's code repository">open_in_new</a></h4>
+    <p align="justify">This is a plugin for the <a href="https://backstage.io/" target="_blank">BackStage Project</a> that provides a seamless way to display <a href="/guides/dora-metrics-four-keys/">DORA Metrics</a> in your developer portals.. <a class="google-material-icons source-available-repo" href="https://github.com/liatrio/backstage-dora-plugin"" target="_blank" aria-label="Liatrio backstage-dora-plugin's code repository">github</a></p>
```

Comment:
Nit: lowercase "p" in "Backstage project"
Nit: remove extra period at end of sentence

Reply:
Fixed in ffc8c4ac41309aaaed1848c58e9e6754dcd91a0c